ema/vuejs/src/main.js

```diff
@@ -0,0 +1,15 @@
+import './assets/main.css'
+
+import { createApp } from 'vue'
+import App from './App.vue'
+import router from './router'
+
+import { registerGlobalComponents } from './components/global-components'
+
+const app = createApp(App)
+
+app.use(router)
+
+registerGlobalComponents(app)
```

Comment:
should we provide this function?

Reply:
I'll guess we'll see when we make EMA out of this